src/Commands/presence.js

```diff
@@ -0,0 +1,15 @@
+const { botErrorHandler } = require('../utils/botErrorHandler');
+try {
+	module.exports = async function (client) {
+		client.user.setPresence({
+			status: 'online',
+			activity: {
+				name: 'cm!help',
+				type: 'LISTENING',
+				url: 'https://github.com/commclassroom/classroom-monitor-bot.',
```

Comment:
I think this dot should be removed.
```suggestion
				url: 'https://github.com/commclassroom/classroom-monitor-bot',
```